tests/queries/0_stateless/02479_mysql_connect_to_self.reference

```diff
@@ -67,6 +67,6 @@ SELECT
     __table1.a AS a,
     __table1.b AS b,
     __table1.c AS c
-FROM mysql(\'127.0.0.1:9004\', \'default\', foo, \'default\', \'\', SETTINGS connection_wait_timeout = 123, connect_timeout = 40123002, read_write_timeout = 40123001, connection_pool_size = 3) AS __table1
+FROM mysql(\'127.0.0.1:9004\', _CAST(\'default\', \'String\'), foo, \'default\', \'\', SETTINGS connection_wait_timeout = 123, connect_timeout = 40123002, read_write_timeout = 40123001, connection_pool_size = 3) AS __table1
```

Comment:
@vdimir what do you think about this change? is it okay?
maybe we need to introduce a new parameter for `EXPLAIN QUERY TREE` to print AST that would be sent to the external databases?

Reply:
Anyway we rewrite `mysql` table function before sending, so `'defailt'` here is just an argument that function, so should be ok.
Another question isn't it possible that some tablefunction won't recognize cast of constant as literal and stop working?